tests/NgTests/Infrastructure/TestLogger.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace NgTests.Infrastructure
+{
+    internal class TestLogger : ILogger
+    {
+        public void Log(LogLevel logLevel, int eventId, object state, Exception exception, Func<object, Exception, string> formatter)
+        {
+            // Console.WriteLine($"{logLevel}: {formatter(state, exception)}");
```

Comment:
A test logger should accept Xunit's `ITestOutputHelper` right?

Reply:
This is existing code I just moved to its own file :) But yes, this should accept Xunit's `ITestOutputHelper`, will fix.

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/2db90127244969eb8cb3e67787811c569b81d4b0